### ISSUE TITLE
Lexend is an alias for Lexend Deca?

### ISSUE THREAD

**Issue by iandinwoodie** [googlefonts/lexend#5]
Based on the following comments:

https://github.com/googlefonts/lexend/blob/20491885ca2cf7ffc556432973e7bdbc701952b5/sources/lexend.yaml#L7-L15

And the following lines from the build script:

https://github.com/googlefonts/lexend/blob/20491885ca2cf7ffc556432973e7bdbc701952b5/sources/build.sh#L12

https://github.com/googlefonts/lexend/blob/20491885ca2cf7ffc556432973e7bdbc701952b5/sources/build.sh#L20

Both Lexend and Lexend Deca have a hyper-expansion factor (`HEXP`) of zero. Is Lexend an alias for Lexend Deca?

**Comment by davelab6** [comment]
@iandinwoodie a fair question. The basic answer is yes, deca is an alias for the default style in Lexend, although from the perspective of a GF user the order of appearance is the other way around. 

The original vision for the project was when Thomas Jockin met Dr Shaver-Troup and proposed to use variable fonts technology to make a highly individualizable, fine tunable, version of the Lexend design that Dr Shaver-Troup had commissioned many years earlier. 

However, since Google Fonts and Docs didn't (and Docs still doesn't) support variable fonts beyond a weight axis, the initial release in the GF API wasn't as a variable font with a hyperextension axis as demonstrated on www.lexend.com, but instead as a set of static fonts. I proposed a set of names with an alphabetical progression so that with an alphabetical sort, like a font picker menu, they get more and more hyperextended. Notably the set started with Deca, leaving room in the API namespace for a family without anything appended - so that eventually the set can be delisted and only the plain family name will be available. 

Then, when the API supported variable fonts, and the Lexend project took the next step with Font Bureau to add a weight axis to the hyperexpansion axis, and to deconstruct that axis into XTRA and SPAC, I decided  (with Dr Shaver-Troup, Font Bureau and font engineer Rosalie Wagner) to upgrade each of the set to a weight axis, and publish a 2 axis vf as "Lexend". 

That's where we are now, and this will likely remain the case until there's more VF support in lots of applications, and the set can be deprecated as planned.
